### PR TITLE
finished javascript-functions

### PR DESCRIPTION
**Pull request by michael5522**
Screen Shot 2022-02-17 at 7 41 42 PM

### CODE REVIEW

File: javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+function getFirstName(person) {
+  var answer = person.firstName;
+  return answer;
+}
+var four = getFirstName({ firstName: 'Lelouche', lastName: 'Lamperouge' });
+console.log('first name: ', four);
+
+function getLastElement(array) {
+  return array[array.length - 2];
```
Comment (BrettAlbright) [inline javascript-functions/main.js]: this should be array.length-1 good sir!

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
one small update!

File: javascript-functions/main.js (hunk 1 above)
Comment (michael5522) [reply javascript-functions/main.js]: Screen Shot 2022-02-17 at 7 58 01 PM
whoops, thanks Brett!

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**